.yamllint

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+# Extend the `default` configuration provided by `yamllint`
+extends: default
+
+# Files to ignore completely
+# 1. All YAML files under directory `node_modules/`, introduced during the Travis run
+# 2. Any SLS files under directory `test/`, which are actually state files
+# 3. Any YAML files under directory `.kitchen/`, introduced during local testing
+ignore: |
+  node_modules/
+  test/**/states/**/*.sls
+  .kitchen/
+  pycharm/osfamilymap.yaml
+  pycharm/files/default/shortcut.desktop.jinja
+  pillar.example
```

Comment:
Unnecessary, will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-b97c55dd898003930e2858fadd3b5c9cL16-L17